dashboard/views.py

```diff
@@ -61,6 +61,8 @@ def get(self, request, username, *args, **kargs):  # pylint: disable=unused-argu
                     status=exc.http_status_code,
                     data={'error': str(exc)}
                 )
+            except Exception:  # pylint: disable=broad-except
```

Comment:
Can we use `except:` here too? Also wouldn't we want this to be re-raised? What kind of exceptions are we expecting here?